docs/releases.md

```diff
@@ -1,62 +1,61 @@
-# Releases
+# Release process
 
-This document describes how the release flow looks like and how to create a Marketplace release.
+This document describes how to create a release of the `marketplaces` repository before publishing it on a given Partner Portal, such as GCP Marketplace.
 
-## Release process
+>**NOTE:** The release process is based on Git tags. Once a tag is created, the release process starts automatically.
 
-The release process is based on Git tags. Once a tag is created, the release process starts automatically.
+## Versioning
 
-### Versioning
+Versioning of `marketplaces` should follow Kyma versions, so the release of `marketplaces` 1.1.1 installs Kyma in version 1.1.1. Versions with a suffix generate pre-releases.
 
-Marketplace versioning should follow Kyma versions. Marketplace in version 1.1.1 installs Kyma in version 1.1.1. Versions with a suffix generate pre-releases.
+## Create a release
 
-### Create a release
+Follow the steps to create a new release.
 
-Follow these steps to create a new Marketplace release:
 >**NOTE:** This instruction is based on the release 1.1.0.
 
-1. Check out the `master` branch with the latest changes:
+1. Clone the `marketplaces` repository on your local machine.
 
     ```bash
-    git checkout master
-    git pull
+    git clone git@github.com:kyma-incubator/marketplaces.git
     ```
 
-2. Create a release branch:
+2. Go to the `marketplaces` folder. If you cloned the repository before, make sure you are on the `master` branch:
 
     ```bash
-   git checkout -b release-1.1
-   git push origin release-1.1
-   ```
+    git checkout master
+    ```
 
-3. Create a Pull Request (PR) with all required changes.
-4. Merge the PR to the release branch.
-5. Check out the release branch:
+3. Fetch and pull the latest changes from the upstream master:
 
     ```bash
-    git checkout release-1.1
+    git fetch --all
     git pull
     ```
 
-6. Create a tag with the proper release version:
+4. Create a release branch:
 
     ```bash
-    git tag 1.1.0
-    ```
+   git checkout -b release-1.1
+   git push upstream release-1.1
```

Comment:
The problem is when you are doing it from scratch (from point 1) then you should push to origin. Same in point 6